src/Microsoft.Azure.SignalR.Management/ServiceManagerOptions.cs

```diff
@@ -1,10 +1,36 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using Microsoft.Azure.SignalR;
+
 namespace Microsoft.Azure.SignalR.Management
 {
     public class ServiceManagerOptions
     {
-        public ServiceTransportType ServiceTransportType { get; set; }
+        public ServiceTransportType ServiceTransportType { get; set; } = ServiceTransportType.Transient;
+
+        public string ConnectionString { get; set; } = null;
+
+        private void ValidateConnectionString()
+        {
+            // if the connection string is invalid, exceptions will be thrown.
+            (string endpoint, string accessKey, string version, int? port) = ConnectionStringParser.Parse(ConnectionString);
```

Comment:
`(_,_,_,_)=` is enough?

Reply:
Yes, the `ConnectionStringParser.Parse` will validate the connection string.

Reply:
No need to use the parsed data, use ` ConnectionParser.Parse(...)` instead